live-examples/js-examples/promise/promise-constructor.html

```diff
@@ -1,9 +1,23 @@
 <pre>
 <code id="static-js">var promise1 = new Promise(function(resolve, reject) {
-  setTimeout(resolve, 100, 'foo');
+  setTimeout(function() { 
+    resolve('foo');
+  }, 100);
```

Comment:
If we want to do this we should bump this a bit so the delay is noticeable (but not annoying). 300 or 400 ms would work better.